src/OVAL/probes/fsdev.c

```diff
@@ -243,7 +243,7 @@ static fsdev_t *__fsdev_init(fsdev_t * lfs, const char **fs, size_t fs_cnt)
 	FILE *fp;
 	size_t i;
 
-	struct mnttab *ment;
+	struct mnttab mentbuf, *ment = &mentbuf;
```

Comment:
Can't we just allocate this on the stack and then use `&mentbuf` for the methods? this aliasing seems unnecessary.

Reply:
Yes for sure.
I just wrote it this way to minimize the code that had to be changed.